lib/alerts/informed_entity.ex

```diff
@@ -29,8 +29,6 @@ defmodule Alerts.InformedEntity do
           | :using_escalator
           | :using_wheelchair
 
-  alias __MODULE__, as: IE
```

Comment:
📓 I just remove this `IE` shortcut wherever I find it. I don't know why it is so hard to type `InformedEntity` and I have seen `InformedEntitySet` also referred to as `IE` so it's better to disambiguate.